Assets/MixedRealityToolkit-Preview/CameraCapture/CameraCaptureARFoundation.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if USE_ARFOUNDATION
```

Comment:
MRTK does not currently have explicit support for ARKit / ARCore. Should this be added when those platforms are officially supported?